### ISSUE TITLE
The results of nuScenes test set

### ISSUE THREAD

**Issue by hht1996ok** [SmartBot-PJLab/MV-JAR#3]
Thank you for your work. What is the effect of MV-JAR in the nuScenes test set？

**Comment by RunsenXu** [comment]
Dear hht1996ok,

Thanks for your interest in our work!

1. We haven't conducted extensive experiments on the nuScenes dataset mainly due to two reasons: 
   - SST does not have an official implementation on nuScenes.
   - We found that SST requires a long time to converge to a reasonable performance, and we didn't have enough computing resources to support experiments on nuScenes.
2. However, we did perform some initial experiments on nuScenes using the following setup:
   - One frame point cloud as input.
   - Training SST for (50 epochs for fine-tuning and scratch; 24 epochs for MV-JAR pre-training).
   - Using the exact same hyperparameters as our experiments on Waymo.
    
    With this setup, the mAP of SST (on the val split, not the test split) improved from 18.48% to 22.08%. Please note that the performance here is not well-tuned and might not represent the best possible outcome.

We hope this information is helpful. Feel free to reach out if you have more questions or need further clarification.

Best regards,
Runsen

**Comment by hht1996ok** [comment]
Dear Runsen,
     Thank you for your reply and I would like to discuss some additional issues with you. I have roughly replicated your paper in  transfusion-L and I found baseline improvement of about 4%+ mAP in the nuscenes val set, however in the test set, the use of pre-training weights resulted in attenuation of network accuracy. 
     In the self-supervised network, we use single-frame data from train+val+test for training, and the same pre-training weights are used in the val set and test set.
     What do you think might be the cause of this problem?
Best regards,
Haotian

**Comment by RunsenXu** [comment]
Hi Haotian,

Did you use train+val+test for MV-JAR pre-training right? What data did you use for fine-tuning?

Best,
Runsen

**Comment by hht1996ok** [comment]
Dear Runsen,

1. Yes, we used train+val+test to train MV-JAR.
2. In val set, we only used train as training data, while in test set, we respectively tried to use train/train+val as training data, and the results were consistent with those described above.

Best regards,
Haotian

**Comment by RunsenXu** [comment]
Hi Haotian,

You mentioned that you use a single frame for pre-training. What about fine-tuning, did you use multi-sweep?
It's possible that different numbers of frames used in pre-training and fine-tuning can cause the performance drop. 

But it's indeed strange that the effects of pre-training are not consistent across val and test sets, especially when you also include the test set for pre-training as well. 

Best,
Runsen

**Comment by hht1996ok** [comment]
Dear Runsen,

1. Yes, we used multi-sweep in fine-turning. To verify the difference between single-sweep and multi-sweep, we are experimenting with pre-training with multi-sweep data, it seems that multi-sweep method does not work badly in val set.
2. This is very strange indeed. In terms of indicators, after using the pre-training model, mAOE decreased to a large extent in val set, but did not change significantly in test set. Maybe the data distribution in train/val was different from that in test and caused the pre-trained network to learn the wrong point cloud geometry? 

Best regards,
Haotian

**Comment by RunsenXu** [comment]
Dear Haotian,

Thank you for sharing your observations. To gain further insights, I would suggest conducting a more detailed analysis of the multi-sweep pre-training approach. Additionally, exploring the use of MV-JAR with temporal information could be a valuable avenue to pursue.

Regarding the inconsistency in performance between the validation and test sets, your hypothesis about different data distributions is plausible. One way to verify this would be to include some of the test data in the fine-tuning process and evaluate the model's performance on the remaining test data. What do you think?

Best regards,
Runsen

**Comment by hht1996ok** [comment]
Dear Runsen,

1. Thank you for answering my question. I'm working on improving the current self-supervise model, and I think it's a good idea to use temporal information for self-supervise model.
2. However, there is no annotation data for the test set in the nuScenes dataset, so your method may not work. Currently I'm trying to fine-tune using only data from the test set. If you are interested, I can share the experimental results with you in the future.

Best regards,
Haotian

**Comment by RunsenXu** [comment]
Dear Haotian,

Please feel free to share the results with me in the future. You can send them to my email and we can discuss them further. I look forward to hearing from you!

Best regards,
Runsen
